app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/NightscoutUploader.java

```diff
@@ -233,6 +240,13 @@ private void populateV1APIMeterReadingEntry(JSONArray array, Calibration record)
             json.put("date", record.timestamp);
             json.put("dateString", format.format(record.timestamp));
             json.put("mbg", record.bg);
+            json.put("xDrip_slope", record.slope);
```

Comment:
this is exactly the thing that will happen. only sent when there is a new calibration, or change in existing one.